puma_joy:
  joy_topic: "joy"
  # --- Params for brake controller --- #
  brake_topic: "puma/brake/command"
  # --- params for dir controller --- #
  dir_topic: "puma/direction/command"
  # --- params for accel puma --- #
  accel_puma_topic: "puma/accelerator/command"
  min_accel: 0
  max_accel: 40 # max value 126
  angle_min_degree: -45
  angle_max_degree: 45
  # --- Params for controller --- #
  lt_left_index: 2
  rt_right_index: 5
  x_left_index: 0
  y_left_index: 1
  x_right_index: 3
  y_right_index: 4
  a_button_index: 0
  b_button_index: 1
  x_button_index: 2
  y_button_index: 3
  lb_button_index: 4
  rb_button_index: 5

puma_controller:
  accelerator_topic: puma/accelerator/command
  brake_topic: puma/brake/command
  reverse_topic: puma/reverse/command
  ackermann_topic: puma/control/ackermann/command
  direction_topic: puma/direction/command
  range_accel_converter: [22, 40]
  connect_to_ackermann_converter: false

# ackermann_converter:
#   cmd_vel_topic: cmd_vel
#   wheel_base: 1.15
#   ackermann_topic: puma/control/ackermann/command
#   arduino_status_topic: puma/arduino/status

